sickbeard/__init__.py

```diff
@@ -286,6 +286,7 @@
 TORRENT_LABEL = ''
 TORRENT_LABEL_ANIME = ''
 TORRENT_VERIFY_CERT = False
+TORRENT_RPCURL = '/transmission"
```

Comment:
@grrr2 I think you shouldn't hardcode this. But set as "transmission" as the default of "torrent_rpcurl" in the UI